pages.de/android/pm.md

```diff
@@ -0,0 +1,24 @@
+# pm
+
+> Zeige Informationen über Apps auf einem Android Gerät.
+> Mehr Informationen: <https://developer.android.com/studio/command-line/adb#pm>.
+
+- Gib eine Liste aller installierten Apps aus:
+
+`pm list packages`
+
+- Gib eine Liste aller installierten System-Apps aus:
+
+`pm list packages -s`
+
+- Gib eine liste aller installierten 3rd-Party Apps aus:
```

Comment:
```suggestion
- Gib eine liste aller installierten Drittpartei-Apps aus:
```

Reply:
This sounds really strange. How about `Drittanbieter`?

Reply:
Yeah, that sounds better. Upon consideration, 3rd-Party-Apps or Third-Party-Apps might also work; after all, there's nothing inherently wrong about anglicisms.